calendarsubscription/calendarsubscription.go

```diff
@@ -14,6 +14,7 @@ type CalendarSubscription struct {
 	UserID     string
 	ScheduleID string
 	LastAccess time.Time
+	CreatedAt  time.Time
```

Comment:
We can leave this out of the struct, we only need it during Authorize as part of the query